### DISCUSSION TITLE
How to stretch tabs to fill tab bar?

### DISCUSSION THREAD

**Discussion by manojkarthick** [wezterm/wezterm#1834] [Q&A]
Discovered wezterm recently and it's awesome! Thanks so much for building this 💯 

I looked at the docs and issues, but I couldn't find a way to stretch tabs to fill the tab bar.

For example, in iTerm2 the default tab bar looks like this:

![Screen Shot 59](https://user-images.githubusercontent.com/7802795/162187459-0257861f-c15e-4421-aed5-626b801f35c2.png)

After setting the "Stretch tabs to fill bar" option,
![Screen Shot 57](https://user-images.githubusercontent.com/7802795/162187444-3c68c23e-c476-4968-bd84-36b37fc7fbec.png)

It transforms to something like this:
![Screen Shot 60](https://user-images.githubusercontent.com/7802795/162187474-7f56867e-f23c-4095-a84a-8e3a45375a74.png)

Is it possible to do something similar in wezterm?

Thank you!

**Comment by wez** [comment]
It's not possible today, but https://github.com/wez/wezterm/issues/1914 is now tracking adding support for this

**Comment by jacksongoode** [comment]
See https://github.com/wez/wezterm/pull/4403
